code-matlab/example_workflows/WORKFLOW_PlotOrderedRaster.m

```diff
@@ -30,8 +30,8 @@
 expCond(1).t = metadata.taskvars.trial_iv_L; % previously stored trial start and end times for left trials
 expCond(2).t = metadata.taskvars.trial_iv_R; 
 
-expCond(1).coord = metadata.coord.coordL; % previously user input idealized linear track
-expCond(2).coord = metadata.coord.coordR; % note, this is in units of "camera pixels", not cm
+expCond(1).coord = metadata.coord.coordL.coord; % previously user input idealized linear track
```

Comment:
this seems to require updated metadata files -- if that's right, how would we do that? we'd need to update the actual data that's on the database, DataLad, etc?